video/out/vo_rpi.c

```diff
@@ -804,6 +804,10 @@ static void destroy_dispmanx(struct vo *vo)
     disable_renderer(vo);
     destroy_overlays(vo);
 
+    if (p->update)
+        vc_dispmanx_update_submit_sync(p->update);
```

Comment:
It looks like the `uninit` also calls this function. Since you're putting this in `destroy_dispmanx` (which is also called by `uninit`), I think that bit can be removed since with this change the `p->update` will always be 0.

Reply:
Good point. Removed the redundant `vc_dispmanx_update_submit_sync`